src/services/ContentService.js

```diff
@@ -53,14 +53,16 @@ define(["structures/ContentCreateStruct", "structures/ContentUpdateStruct", "str
      * @method loadRoot
      * @param callback {Function} callback executed after performing the request (see
      * {{#crossLink "ContentService"}}Note on the callbacks usage{{/crossLink}} for more info)
+     * @param xhrCallbacks {Object} a set of callbacks to apply on a specific XHR event like onload, onerror, onprogress, etc.
      */
-    ContentService.prototype.loadRoot = function (callback) {
+    ContentService.prototype.loadRoot = function (callback, xhrCallbacks) {
```

Comment:
I'll add it to `.getContentService().createContent()` method then.
Other methods will be updated when needed, as you said.